### DISCUSSION TITLE
Feature request: Feature to start Tasmota OTA update progress via TDM

### DISCUSSION THREAD

**Discussion by miXza81** [jziolkowski/tdm#113] [Ideas]
How about Menu Item that will start Tasmota OTA update progress (for single Tasmota device) via Tasmota Device Manager GUI?

**Comment by jziolkowski** [comment]
It was included in some previous release, and TBH I don't remember why I disabled it :/

**Comment by miXza81** [comment]
Ok. Sad to hear.

**Reply by jziolkowski** [reply]
Nah, don't worry. I'll make it work again.

**Comment by miXza81** [comment]
Not so sad to hear. :-) Thanks!
